testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/Kafka.java

```diff
@@ -50,6 +59,12 @@ public void extendEnvironment(Environment.Builder builder)
         builder.addContainers(createZookeeper(), createKafka(), createSchemaRegistry())
                 .containerDependsOn(KAFKA, ZOOKEEPER)
                 .containerDependsOn(SCHEMA_REGISTRY, KAFKA);
+
+        MountableFile logConfigFile = forHostPath(dockerFiles.getDockerFilesHostDirectory("common/kafka").getPath("log.properties"));
```

Comment:
nit: extract copying of the file to a method?

Reply:
Another approach is:
```
        builder.configureContainers(dockerContainer -> {
            if (isPrestoContainer(dockerContainer.getLogicalName())) {
                dockerContainer
                        .withCopyFileToContainer(
                                forHostPath(configDir.getPath("log.properties")),
                                CONTAINER_PRESTO_ETC + "/log.properties");
            }
        });